ur_robot_driver/doc/controllers.md

```diff
@@ -0,0 +1,182 @@
+stops, e.g. when leaving the robot's operation space or forcing it to change its configuration.
+
+---
+Controllers below this line are probably less useful for the majority of applications, but can be
+used nevertheless.
+
+### `pos_joint_traj_controller`
+**Type:** [`position_controllers/JointTrajectoryController`](http://wiki.ros.org/joint_trajectory_controller)
+
+This is the most commonly used ROS controller implementing the
+[`control_msgs/FollowJointTrajectory`](http://docs.ros.org/en/latest/api/control_msgs/html/action/FollowJointTrajectory.html)
+interface. For controlling the robot it will send joint position commands.
+
+**We do not recommend using this controller, as it does not respect the robot's speed scaling value.
+Setting the speed slider at another position than 100% or pausing the robot will lead to unexpected
+side-effects! Use the scaled version of this controller instead.**
+
+### `vel_joint_traj_controller`
+**Type:** [`velocity_controllers/JointTrajectoryController`](http://wiki.ros.org/joint_trajectory_controller)
+
+Implements the
+[`control_msgs/FollowJointTrajectory`](http://docs.ros.org/en/latest/api/control_msgs/html/action/FollowJointTrajectory.html)
+interface. For controlling the robot it will send joint velocity commands.
+
+As this controller uses closed-loop velocity control, it requires tuned PID parameters. Currently,
+they are not optimized, see
+[#119](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/issues/119).
+
+### `scaled_vel_joint_traj_controller`
+**Type:** [`velocity_controllers/ScaledJointTrajectoryController`](http://wiki.ros.org/scaled_joint_trajectory_controller)
+
+Similar to the non-scaled version it implements the
+[`control_msgs/FollowJointTrajectory`](http://docs.ros.org/en/latest/api/control_msgs/html/action/FollowJointTrajectory.html)
+interface. For controlling the robot it will send joint velocity commands.
+
+In contras to the non-scaled version, the speed slider position or any other speed-modifying action
+such as pausing, EM-Stop or speed scaling due to limits configured on the robot will be respected by
+this controller.  If a trajectory is given with time constraints and the trajectory is slowed down
+during execution, the action fails. If no time constraints are set and the motion is slowed down
+during execution, the actual action call will succeed.
+
+### `forward_joint_traj_controller`
+**Type:** [`pass_through_controllers/JointTrajectoryController`](http://wiki.ros.org/pass_through_controllers)
+
+This controller implements the
+[`control_msgs/FollowJointTrajectory`](http://docs.ros.org/en/latest/api/control_msgs/html/action/FollowJointTrajectory.html)
+interface. Instead of interpolating the trajectory on ROS side, this forwards the complete
+trajectory to the robot for execution leaving the robot in charge of calculating accelerations and
+speeds. Therefore, this will lead to motions most similar to what you could configure on the teach
+pendant.
```

Comment:
Should this say something about possible divergence from the commanded trajectory due to the OEM controller not sticking to the setpoints in that trajectory but "doing its own thing'?

Reply:
Thought about it when starting the paragraph, did forget it while writing the paragraph... Added in fb6c3ee